### ISSUE TITLE
Excuse me,I do not know why I got the output by running the code like this:

### ISSUE THREAD

**Issue by hhhhhhogan** [pytorch/pytorch#61950]
Excuse me,I do not know why I got the output by running the code like this:
![image](https://user-images.githubusercontent.com/40531744/126470122-07263c08-b7c1-4fc3-a680-9f9720fc963f.png)
Why the leaf node of a ,c is true, and  does the same operation  in b, d, but the result of the leaf node of b d are different?

**Comment by kshitij12345** [comment]
> All Tensors that have requires_grad which is False will be leaf Tensors by convention.

https://pytorch.org/docs/stable/generated/torch.Tensor.is_leaf.html

**Comment by albanD** [comment]
Hi,

We use github issues only for bugs or feature requests.
Please use the forum to ask questions: https://discuss.pytorch.org/

In your case, since the second Tensor doesn't require gradients, it is always a leaf as mentioned in the doc linked by @kshitij12345
